grammars/php.cson

```diff
@@ -470,15 +470,8 @@
   'heredoc':
     'patterns': [
       {
-        'begin': '(?=<<<\\s*("?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)\\s*$)'
+        'begin': '(?=<<<\\s*("?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)[ \\t]*$)'
```

Comment:
I highly doubt that as the begin pattern can only match a single line.  If you change it, do the specs still pass?